runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -92,13 +92,6 @@ TR_J9ServerVM::isSameOrSuperClass(J9Class *superClass, J9Class *subClass)
    if (superClass == subClass) // same class
       return true;
 
-   void *superClassLoader, *subClassLoader;
-   JITServer::ServerStream *stream = _compInfoPT->getMethodBeingCompiled()->_stream;
-   JITServerHelpers::getAndCacheRAMClassInfo(superClass, _compInfoPT->getClientData(), stream, JITServerHelpers::CLASSINFO_CLASS_LOADER, &superClassLoader);
-   JITServerHelpers::getAndCacheRAMClassInfo(subClass, _compInfoPT->getClientData(), stream, JITServerHelpers::CLASSINFO_CLASS_LOADER, &subClassLoader);
-   if (superClassLoader != subClassLoader)
-      return false;
```

Comment:
@mpirvu do you know the jitserver version checked the classloaders when the base implementation does not?

Reply:
The implementation was supposed to reduce the number of messages. However, if the assumption is based on (classes loaded by different class loaders cannot be related) is invalid, then it must be removed.